src/UI/templates/default/Deck/deck.less

```diff
@@ -21,3 +21,23 @@
     }
   }
 }
+
+@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
+  .row-eq-height {
+
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
```

Comment:
those first four lines are could be shared with @screen-md-min in a sm-min section, to enhance readability.